### DISCUSSION TITLE
Generate tag-based documentation for aiosmtpd.readthedocs

### DISCUSSION THREAD

**Discussion by pepoluan** [aio-libs/aiosmtpd#360] [General]
Hi @warsaw & @asvetlov ,

Can we generate a version-pinned (or tag-pinned) documentation for aiosmtpd on readthedocs?

Currently it has only `latest` and `stable`, we should have snapshots by tag. At the very least, we should have a snapshot of the documentation with `1.4.3` tag.

(I'm not listed as one of the maintainers there so I can't generate a pinned tag-based doc myself.)

![image](https://user-images.githubusercontent.com/642259/212293048-df558208-19f1-49ec-aa69-e12bbcd35bec.png)

**Comment by pepoluan** [comment]
Any updates on this @warsaw @asvetlov ?

## Edit:

As a comparison, this is how `aiosmtpd-pepoluan` -- which tracks my fork -- looks like:

![image](https://user-images.githubusercontent.com/642259/212293489-a4176f4c-9535-47aa-b8f0-032f8ad86f0b.png)

**Comment by warsaw** [comment]
Hi @pepoluan - I just sent you an invite as maintainer on the RTD project, assuming your RTD user id is the same as your GH.  LMK if this works for you, and thanks!

**Reply by pepoluan** [reply]
Okay I'm currently out and about... will check when I get home. Thanks!

**Reply by pepoluan** [reply]
Okay, I've verified I'm in!

Thanks @warsaw !

**Reply by webknjaz** [reply]
@pepoluan if you want to wire a CNAME `aiosmtpd.aio-libs.org` to RTD — let me know!
